run-tests.py

```diff
@@ -44,6 +44,11 @@ def _build(image, salt=False):
             b"ADD {0} /srv/formula/{0}\n"
             b"RUN salt-call -l debug state.highstate\n"
         ).format(_formula)
+        if image in ("centos7",):
+            # Salt fail to enable a systemd service if systemd is not running
+            # (during the docker build phase)
+            # This is a workaround.
+            dockerfile_content += b"RUN systemctl enable supervisord\n"
```

Comment:
This issue is really annoying (systemctl is-enabled require a dbus connection wich systemd). An other fix could be to run ``state.highstate`` in a running container.(docker run, exec salt-call, docker stop, docker commit), but the complexity introduced seems too high VS the quick workaround.